### ISSUE TITLE
Why is there no "Every 6h" option in the Agent Schedule?

### ISSUE THREAD

**Issue by level20peon** [cantino/huginn#1104]
And: Would it work if I set the schedule of 6h manually inside the database?

**Comment by cantino** [comment]
No, you'd have to add it to agent.rb in `SCHEDULES` and huginn_scheduler.rb as well.

**Comment by knu** [comment]
We could probably make it configurable via .env.

**Comment by level20peon** [comment]
> We could probably make it configurable via .env.

Please elaborate.

**Comment by cantino** [comment]
@knu, right now we filter / validate the list when importing Scenarios, so we'd want to avoid people removing schedules locally and having imported data be invalid.

**Comment by knu** [comment]
@cantino You're right!  They could even remove a schedule that is used by some of their agents by mistake, rendering them to never be run without noticing.
